.gitignore

```
@@ -23,3 +23,4 @@ dist-ssr
 *.sln
 *.sw?
 envoy.yaml
+.env
```

I suggest ignoring all dot-prefixed files except for important ones (e.g., this `.gitignore` file itself):
```suggestion
.*
!.gitignore
```

Other dot-prefixed files like `.vscode/*`, `.idea`, etc., may no longer need to be specified.